src/basic/hashmap.c

```diff
@@ -254,7 +254,7 @@ struct hashmap_type_info {
         unsigned n_direct_buckets;
 };
 
-static const struct hashmap_type_info hashmap_type_info[_HASHMAP_TYPE_MAX] = {
+const struct hashmap_type_info hashmap_type_info[_HASHMAP_TYPE_MAX] = {
```

Comment:
hmm, shouldn't this be marked with `_unused_`?

Also, how come the compiler doesn't complain that this doesn't have a prototype if not static? Weird?

Reply:
`_unused_` seems to be the opposite — do not warn if the variable is not used (at least according to the docs). It seems that `_used_` DTRT. I'll switch to that.

The compiler doesn't warn because it's a variable, not a function.